sycl/test-e2e/Reduction/reduction_span_pack.cpp

```diff
@@ -6,7 +6,7 @@
 
 // Windows doesn't yet have full shutdown().
 // UNSUPPORTED: ze_debug && windows
-
+// REQUIRES: aspect-usm_shared_allocations
```

Comment:
Same

Reply:
These tests that involve spans seem unfriendly to buffers/accessor as there are no constructors that accept accessors as arguments. I could probably use the get_ptr method on an accessor and pass that pointer to the span but I've got a feeling that's probably undefined behavior, feel free to correct me though!